src/Axiom/Extensionality/Heterogeneous.agda

```diff
@@ -9,10 +9,12 @@
 module Axiom.Extensionality.Heterogeneous where
 
 import Axiom.Extensionality.Propositional as P
+  using (Extensionality; lower-extensionality)
```

Comment:
Inconsistency with your style of `using` on one line, and the list of used names on another below. I think what you have on L12 is 'better', see below.